### DISCUSSION TITLE
Back Office - Add logout button

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#125] [Q&A] [answered]
Hi guys,
how can I add the "Logged User" section in the top left corner of my Back Office, with the logout button?
<img width="264" alt="Screenshot 2022-11-30 alle 11 34 22" src="https://user-images.githubusercontent.com/28152586/204773708-02c9726a-979a-4b66-874a-f91e8fc3e24c.png">

I didn't found it on the documentation.

Thanks.

**Comment by peppemanzi** [comment, marked as answer]
I think you need to implement [this](https://micro-lc.io/documentation/docs/micro-lc/authentication#userinfourl) configuration in order to have the user menu shown.

**Reply by peppemanzi** [reply]
Please note that the endpoints that you are referring to in the configuration must be exposed through the `api-gateway`, since they are called by the FE application.

**Reply by frengooo** [reply]
Thanks @peppemanzi, I was missing the `/userinfo` and the `/logout` endpoints.
I'll update the documentation.
